racket-cas/racket-cas.rkt

```diff
@@ -2868,6 +3230,74 @@
       [(⊖ u)  #f]
       [_      #t])) ; todo: add cases for variables with assumptions
 
+(define (solve-by-inverse-stacks w x)
```

Comment:
Hi @soegaard , do you have any suggestions on how to share codes between `solve-by-inverse-stacks` and `solve-by-inverse`?

Two differences from `solve-by-inverse`:

1. `solve-by-inverse-stacks` needs to return the list of all intermediate results/eqns.
2. the `let/ec return`.

Reply:
THe function has been removed.